src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {

```

Comment:
Don't use static methods in your solution. Even though there are no static methods in this class, ensure that you don't use them elsewhere in your project as well.